QA.md

```diff
@@ -8,7 +8,7 @@ In this interface, Batman must be able to tell the Batcave's supercomputer the l
 
 You are requested to create acceptance tests scenarios to validate if coordinates sent by batmobile GPS are returning precise information about villains within the [Gotham boundaries](https://gist.githubusercontent.com/pitteri/d56780d610cb8e0a43bfa94fc54b71cd/raw/dcdd965c84cd05d856ae32646be69868d4a80afa/gotham_bbox.json).
 
-> Automated tests should be created by using using [Rest Assured](http://rest-assured.io/) or [Cucumber for Java](https://cucumber.io/)
+> Automated tests should be created by using [Rest Assured](http://rest-assured.io/) or [Cucumber for Java](https://cucumber.io/)
```

Comment:
On the base Branch, I have reformated this line to use bullet points.
See the merge conflicts on your branch. -https://github.com/rburrowes/code-challenge/pull/3/conflicts